assets/chezmoi.io/docs/reference/command-line-flags/common.md

```diff
@@ -29,19 +40,11 @@ Types can be preceded with `no` to remove them.
 
 Types can be explicitly excluded with the `--exclude` flag.
 
-!!! example
-
-    `--include=files` specifies all files.
-
 ## `--init`
 
 Regenerate and reread the config file from the config file template before
 computing the target state.
 
-## `--interactive`
```

Comment:
Again I refer to https://gist.github.com/KapJI/049b54326d0aec676b3ea49c573fa92a

It's listed within global flags.

Reply:
Addressed on `--help`. It's global because it's easier in the code, but it doesn't apply to *most* commands. It won't cause an *error* because it's global, but that doesn't mean it's meaningful outside of the commands that care about `--interactive`.